packages/feedback-service/src/e2e/feedback.spec.ts

```diff
@@ -0,0 +1,151 @@
+import dotenv from 'dotenv';
```

Comment:
I don't see `dotenv` being used anywhere in this file. You can remove this import.